std_msgs/Header header
int32 robot_id
float64[] distances
float64[] angles